### PR TITLE
Add pagination 

### PR DESCRIPTION
**Pull request by georgeschena**
WHAT: Added pagination 

WHY: To allow easier navigation of tenancies

Screenshot 2019-09-23 at 11 05 12

### CODE REVIEW

File: app/controllers/tenancies_controller.rb [hunk 1]
```diff
@@ -14,6 +14,8 @@ def index
     @number_of_pages = response.number_of_pages
     @user_assigned_tenancies = valid_tenancies(response.tenancies)
     @showing_paused_tenancies = response.paused
+
+    @tenancies = Kaminari.paginate_array(@user_assigned_tenancies).page(params[:page]).per(10)
```
Comment (elena-vi) [inline app/controllers/tenancies_controller.rb]: are you sure this is definitely 10 per page?
I think this might be defined somewhere else

File: app/controllers/search_tenancies_controller.rb [hunk 2]
```diff
@@ -17,6 +17,8 @@ def show
       post_code: @post_code,
       tenancy_ref: @tenancy_ref
     )
+    @number_of_pages = @results.dig(:number_of_pages)
+    @tenancies = Kaminari.paginate_array(@results.dig(:tenancies)).page(params[:page]).per(20)
```
Comment (elena-vi) [inline app/controllers/search_tenancies_controller.rb]: are you sure this is definitely 20 per page?
I think this might be defined somewhere else

Comment (kjdchapman) [inline app/controllers/search_tenancies_controller.rb]: Is this worth splitting up into multiple lines? I found it hard to parse.

```suggestion
    tenancies = @results.dig(:tenancies)
    page = params[:page]
    @tenancies = Kaminari.paginate_array(tenancies).page().per(20)
```

File: app/views/common/_pagination.html.erb [hunk 3]
```diff
@@ -1,18 +1,7 @@
...
-    <div class="column-one-third column-align-center">
-      <span class="pagination__description">Page <%= page_number %> of <%= number_of_pages %></span>
-    </div>
-    <div class="column-one-third column-align-right">
-      <% next_link_href = page_number == number_of_pages ? 'javascript:;' : next_page_path %>
-      <% next_link_class = page_number == number_of_pages ? 'link--disabled link--forward' : 'link--forward' %>
-
-      <%= link_to('Next', next_link_href, class: next_link_class) %>
+    <div class="column-align-center">
+      <%= paginate @tenancies, :total_pages => @number_of_pages %>
```
Comment (kjdchapman) [inline app/views/common/_pagination.html.erb]: image
:ok_hand:

Comment (kjdchapman) [reply app/views/common/_pagination.html.erb]: Oh, could use terser hash syntax here:
```suggestion
      <%= paginate @tenancies, total_pages: @number_of_pages %>
```

File: app/controllers/search_tenancies_controller.rb (hunk 2 above)
Comment (georgeschena) [reply app/controllers/search_tenancies_controller.rb]: Yes you're correct, I have removed it from the variable and let me be handled elsewhere. Good spot.

File: app/controllers/tenancies_controller.rb (hunk 1 above)
Comment (georgeschena) [reply app/controllers/tenancies_controller.rb]: Yes you're correct, I have removed it from the variable and let me be handled elsewhere. Good spot.

File: app/controllers/search_tenancies_controller.rb (hunk 2 above)
Comment (georgeschena) [reply app/controllers/search_tenancies_controller.rb]: I feel it reads fairly well now, and I don't want to clutter the controller with more variables.
